dotnet/src/Connectors/Connectors.AzureOpenAI/Core/ClientCore.File.cs

```diff
@@ -1,44 +1,81 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
-/* 
-Phase 05
-- Ignoring the specific Purposes not implemented by current FileService.
-*/
-
 using System;
 using System.ClientModel;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using Azure.AI.OpenAI;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
 using OpenAI.Files;
 
-using OAIFilePurpose = OpenAI.Files.OpenAIFilePurpose;
-using SKFilePurpose = Microsoft.SemanticKernel.Connectors.OpenAI.OpenAIFilePurpose;
-
 namespace Microsoft.SemanticKernel.Connectors.AzureOpenAI;
 
 /// <summary>
-/// Base class for AI clients that provides common functionality for interacting with OpenAI services.
+/// Base class for AI clients that provides common functionality for interacting with Azure OpenAI services.
 /// </summary>
 internal partial class ClientCore
 {
     /// <summary>
-    /// Uploads a file to OpenAI.
+    /// Initializes a new instance of the <see cref="ClientCore"/> class.
+    /// </summary>
+    /// <param name="endpoint">Azure OpenAI deployment URL, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
+    /// <param name="apiKey">Azure OpenAI API key, see https://learn.microsoft.com/azure/cognitive-services/openai/quickstart</param>
+    /// <param name="httpClient">Custom <see cref="HttpClient"/> for HTTP requests.</param>
+    /// <param name="logger">The <see cref="ILoggerFactory"/> to use for logging. If null, no logging will be performed.</param>
+    internal ClientCore(
+        string endpoint,
+        string apiKey,
+        HttpClient? httpClient = null,
+        ILogger? logger = null)
```

Comment:
Should all constructors be in `ClientCore.cs`

Reply:
Yeah, that is expected as this is a core we should not have specific constructor for a specific modality.

Reply:
Fixed